docs/theory/sediment.md

```diff
@@ -1,312 +1,354 @@
 # Sediment Transport
 
-## 2-D H-R equation
-The 2-D H-R equations are
+Our treatment of the transport of sediment is based on a model developed by
+Hairsine and Rose that accounts for size-selective sediment transport using a
+particle size distribution.
 
-\begin{equation}
-\label{eqn:hr2d}
-\frac{\partial hc_{i}}{\partial t}+\frac{\partial uhc_{i}}{\partial x} + \frac{\partial vhc_{i}}{\partial y} = e_{i} + e_{ri} + r_{i} + r_{ri} - d_{i},
-\end{equation}
+## 2-D Hairsine-Rose (H-R) Equations
 
-\begin{equation}
-\frac{\partial M_{i}}{\partial t} = d_{i} - e_{ri} - r_{ri},
-\end{equation}
+The H-R equations use a particle size distribution consisting of a set of
+$I$ discrete particle/sediment size classes $i = 1, 2, ..., I$. Each size class
```

Comment:
We used $i$ to represent a grid cell, so how about we use a different symbol for the sediment size class (say $k$)?

Reply:
Sure. In the literature, these $i$, $j$, $k$ indices are often viewed as interchangeable, since one often has to form expressions with several of them, including summations, etc. But your suggestion does add clarity in this case. If we wanted to create a size-class-specific index, we might even use something like $s$.